apps/aecore/test/aecore_suite_utils.erl

```diff
@@ -229,6 +230,16 @@ mine_blocks(Node, NumBlocksToMine, MiningRate, Type) ->
     end.
 
 
+mine_all_txs(Node) ->
+    MaxBlocks = 50,
+    aecore_suite_utils:subscribe(Node, block_created),
```

Comment:
```suggestion
```
Is this not done in `mine_blocks_until_txs_on_chain` hence making this redundant - and potentially leading to an unwanted extra subscription?